df.history.xml

```diff
@@ -3227,34 +3224,34 @@
     </class-type>
 
     <class-type type-name='history_event_collection_performancest' inherits-from='history_event_collection'>
-        <int32_t/>
+        <int32_t name='parent_collection' ref-target='history_event_collection' comment="all seen were occasions"/>
         <int32_t name='civ' ref-target='historical_entity'/>
-        <int32_t/>
-        <int32_t/>
+        <int32_t comment="0-11 seen"/>
+        <int32_t comment="0-9 seen"/>
         <int32_t name='ordinal'/>
     </class-type>
 
     <class-type type-name='history_event_collection_competitionst' inherits-from='history_event_collection'>
-        <int32_t/>
+        <int32_t name='parent_collection' ref-target='history_event_collection' comment="all seen were occasions"/>
         <int32_t name='civ' ref-target='historical_entity'/>
-        <int32_t/>
-        <int32_t/>
+        <int32_t comment="0-13 seen"/>
```

Comment:
Another case where an enum might be useful (although admittedly tedious to type out), although it could also be an ID of something that doesn't have very many instances in the world where you were testing.

Reply:
Again, I want some indication other than just a small range to make it an enum.

Reply:
Yup, realized that in the second part of my comment. I think your comments here are useful regardless of what types the fields end up being, so feel free to keep adding those